README.md

```diff
@@ -49,6 +54,10 @@ An attempt at collecting some of the collections of accessible components but al
 
 - [Responsive Table](http://adrianroselli.com/2017/11/a-responsive-accessible-table.html) - A full tutorial for a responsive, accessible table. [Table demo here](https://codepen.io/aardrian/pen/YEKmxP?editors=1000)
 
+## Tabs
+
+- [a11y-tabs](https://github.com/agnosticui/a11y-tabs) -  A lightweight (<1Kb) JavaScript package to facilitate a11y-compliant tabbed interfaces.
```

Comment:
Good suggestion! Lets add this.

Reply:
I'll update the PR per your feedback next chance I get thanks @Kolombiken